src/Centreon/Domain/Monitoring/ResourceGroup.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Centreon\Domain\Monitoring;
+
+/**
+ * Resource group model for resource repository
+ *
+ * @package Centreon\Domain\Monitoring
+ */
+class ResourceGroup
+{
+    /**
+     * Id of the resource group
+     *
+     * @var int
+     */
+    private $id;
+
+    /**
+     * Name of the resource group
+     *
+     * @var string
+     */
+    private $name;
+
+
+    /**
+     * Get resource group id.
+     *
+     * @return integer
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * ResourceGroup constructor
+     */
+    public function __construct(int $resourceGroupId, string $resourceGroupName)
+    {
+        $this->id = $resourceGroupId;
+        $this->name = $resourceGroupName;
+    }
```

Comment:
Could you switch this method before getId()  ? And add @param doc block.

Reply:
Yep done :+1: